lib/internal_metric/global.rb

```diff
@@ -141,5 +146,15 @@ def sidekiq_paused_states
 
       paused
     end
+
+    def missing_uploads(type)
+      missing = {}
+
+      RailsMultisite::ConnectionManagement.each_connection do |db|
+        missing[{ db: db }] = Discourse.stats.get("missing_#{type}_uploads") || 0
```

Comment:
I think this is going to create a metric for every single site which bloats Prometheus. Probably just set the default to `nil` for now which we will skip the metric.

Reply:
it's done.